chaos/cluster/cluster.go

```diff
@@ -21,11 +21,24 @@ import (
 )
 
 const (
-	// We sleep 10 seconds to give some time for ssh command to cleanly finish.
-	cmdReboot          = "nohup sh -c 'sleep 10 && sudo ifconfig eth0 down && sudo systemctl stop kubelet && sudo reboot' >/dev/null 2>&1 &"
-	cmdRebootWithSleep = "nohup sh -c 'sleep 10 && sudo ifconfig eth0 down && sudo systemctl stop kubelet && sleep %ds && sudo reboot' >/dev/null 2>&1 &"
+	stallServiceTmpPath = "/tmp/stall.service"
+	stallServiceTpl     = `
+[Unit]
+Description=reboot stall
+DefaultDependencies=no
+Before=sysinit.target
 
-	cmdSystemUp = "sudo systemctl is-active kubelet"
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/sleep %ds
+
+[Install]
+WantedBy=multi-user.target
+`
+	cmdEnableStallService  = "sudo mv /tmp/stall.service /etc/systemd/system/stall.service && sudo systemctl daemon-reload && sudo systemctl enable stall.service"
+	cmdDisableStallService = "sudo systemctl disable stall.service && sudo mv /etc/systemd/system/stall.service /tmp/stall.service && sudo systemctl daemon-reload"
```

Comment:
It's fine to leave the service in its location under `/etc`. `systemctl disable` will be sufficient.